osi_lane.proto

```diff
@@ -817,6 +816,45 @@ message LaneBoundary
         // See \c LaneBoundary .
         //
         optional double height = 3;
+
+        // Alternation of dashes in case of a dashed lane boundary. In
+        // context, this field gives information about the location of
+        // dashes on the boundary line.
+        //
+        optional Dash dash = 4;
+
+        // This enum describes the alternation of dashes in case of a
+        // dashed lane boundary.
+        //
+        // \note The enum descriptions adhere to the definition direction
+        // of the lane boundary points, meaning that start or end of a dash
```

Comment:
[...] of the lane boundary points. This means that start or end of a dash [...]

Reply:
I'll push this...